### PR TITLE
Loader - Changes

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Reloaded loader
General changes, atmost working in all rooms

## Why / Balance
Baseline

## Technical details
Mapping

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame,

image

image

## Breaking changes
N/A

**Changelog**
N/A

### CODE REVIEW

File: Resources/Maps/Shuttles/loader.yml [hunk 1]
```diff
@@ -12,28 +13,27 @@ entities:
   entities:
   - uid: 1
     components:
-    - name: NT14 Loader PR-712
+    - name: grid
```
Comment (TsjipTsjip) [inline Resources/Maps/Shuttles/loader.yml:16]: Not sure how this happens, but this is suspicious. How did you save the grid? Which command did you use?

File: Resources/Maps/Shuttles/loader.yml
Comment (TsjipTsjip) [inline Resources/Maps/Shuttles/loader.yml:1]: Couple of issues:
- SMES bank is not connected to the substation with Hv cabling.
- I'm relatively sure `fixgridatmos` still needs to be run. Not 100% tho.

Rest looks good to me. Merging this closes #720 .

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]

File: Resources/Maps/Shuttles/loader.yml (hunk 1 above)
Comment (dvir001) [reply Resources/Maps/Shuttles/loader.yml:16]: The file was a map to start with, this is why its like this
This is a rebuild of the ship into grid from 0

Comment (TsjipTsjip) [reply Resources/Maps/Shuttles/loader.yml:16]: Yeeep, that would make sense... It's overridden by the shipyard system anyways on init, so this is fine. Resolved.

**Review by TsjipTsjip — APPROVED** [review]
Tested in-game, this works. Make sure to close #720 after merge.

**Review by MagnusCrowe — APPROVED** [review]

**Comment by Estataa** [comment]
These changes broke the ship. the power DOES NOT WORK :(
image

**Pull request merged**